docs/vwii/vwii-modding.md

```diff
@@ -2,23 +2,21 @@
 ---
 This page will guide you through the process of modding your vWii.
 
-We will now place the required Homebrew files on the SD Card.
-
 ?> **Notice**
     Your SD Card will need to be formatted as FAT32. If your SD Card is not formatted to FAT32, use [GUIFormat](http://ridgecrop.co.uk/index.htm?guiformat.htm) with 32k (32768) Allocation unit size to format it. **Do not** label the SD Card as `wiiu` or it will cause issues with homebrew.
     
 ?> If you have hacked your Wii U in the past, you can use the same SD Card for this process.
-    
-    
 
 ### What You Need {docsify-ignore}
 
-- The latest files from [Tiramisu for your café](https://tiramisu.foryour.cafe).
-    - Click on `Download Tiramisu`.
-    - If you have already installed Tiramisu, you do not need to redownload it.
-- The latest release of the [vwii-compat-installer](https://github.com/TheLordScruffy/vwii-compat-installer/releases).
-- The <a href="docs/files/Patched_IOS80_Installer_for_vWii.zip" download>Patched IOS 80 Installer for vWii</a>. ([Source](https://github.com/Lazr1026/Patched-IOS80-Installer-for-vWii))
-- The <a href ="docs/files/d2x_cIOS_Installer.zip" download>d2x cIOS Installer</a>.
+?> If youre using Tiramisu, this tutorial will still work, simply skip the Aroma download step when reading the instructions.
+
+- The latest version of [Aroma](https://aroma.foryour.cafe)
+    - Click on `Download Tiramisu`

```

Comment:
You are having the guide say to download Aroma but then the instructions for it say to "Download Tiramisu".